lib/loaders/entry-loader.js

```diff
@@ -1,7 +1,9 @@
 const path = require('path');
 const loaderUtils = require('loader-utils');
-const apiPath = path.join(__dirname, 'api.js');
-
+var apiPath = path.join(__dirname, 'api.js')
+if(/win32/gi.test(process.platform)){
```

Comment:
We suggest to use `loader-utils.stringifyRequest` to stringify the module path, it will cover more edge cases.

Reply:
Yeah, good advise ! I'll commit the code later.

Reply:
> I would suggest using `loaderUtils.stringifyRequest` to stringify the module path, it will cover more edge cases.

I'd tried to use [loaderUtils.stringifyRequest](https://github.com/webpack/loader-utils),the following code
```
loaderUtils.stringifyRequest(this,path.join(__dirname, 'api.js'))
// return 
// \"e:/a.js\"
```
It seems still can't resolve it.
What's your opinion?

Reply:
Have you tested it in a Windows machine?

Reply:
> Have you tested it in a Windows machine?

yeah,i had tested it in my windows machine.

Reply:
> Have you tested it in a Windows machine?

The transform result on windows is `require('!!"E:/api.js"')` .
`loaderUtils.stringifyRequest` just like `JSON.stringify` to parse the string.

Reply:
> I would suggest using `loaderUtils.stringifyRequest` to stringify the module path, it will cover more edge cases.

Hi,I had tried another way to fix the problem. Look at the commit [fb7a4aa8a38856ab99796a2486d4c44d43af31b4](https://github.com/liximomo/lazy-compile-webpack-plugin/pull/10/commits/fb7a4aa8a38856ab99796a2486d4c44d43af31b4). It works on my windows and mac machine.